<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QGroupBox" name="GroupBox_Serial">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>250</width>
     <height>298</height>
    </rect>
   </property>
   <property name="title">
    <string>USB-to-Serial</string>
   </property>
   <widget class="QPushButton" name="PushButton_PortConnect">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>90</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_PortDisconnect">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>90</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QLabel" name="flowControlLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>190</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Flow control:</string>
    </property>
   </widget>
   <widget class="QLabel" name="dataBitsLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>123</y>
      <width>190</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Data bits:</string>
    </property>
   </widget>
   <widget class="QLabel" name="stopBitsLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>181</y>
      <width>190</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop bits:</string>
    </property>
   </widget>
   <widget class="QLabel" name="parityLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>152</y>
      <width>190</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Parity:</string>
    </property>
   </widget>
   <widget class="QLabel" name="baudRateLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>94</y>
      <width>190</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>BaudRate:</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>370</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_Settings">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>90</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QLabel" name="portNameLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>370</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Port Name:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GroupBox_Move">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>208</y>
     <width>251</width>
     <height>249</height>
    </rect>
   </property>
   <property name="title">
    <string>Move</string>
   </property>
   <widget class="QLineEdit" name="LineEdit_Distance">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_Velocity">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_Accel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_Decel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_Move">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_Abort">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Abort</string>
    </property>
   </widget>
   <widget class="QLabel" name="IPLabel_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance (counts)</string>
    </property>
   </widget>
   <widget class="QLabel" name="IPLabel_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Velocity (counts/sec)</string>
    </property>
   </widget>
   <widget class="QLabel" name="IPLabel_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Accel (counts/sec^2)</string>
    </property>
   </widget>
   <widget class="QLabel" name="IPLabel_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>140</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Decel (counts/sec^2)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GroupBox_Jog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>251</width>
     <height>139</height>
    </rect>
   </property>
   <property name="title">
    <string>Jog</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="IPLabel_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>150</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Jog Velocity (counts/sec)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Positive</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Negative</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Activate Jog</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GroupBox_Home">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>398</y>
     <width>251</width>
     <height>59</height>
    </rect>
   </property>
   <property name="title">
    <string>Home</string>
   </property>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Abort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="IPLabel_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Position</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_Position">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>370</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="GroupBox_Limits">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>468</y>
     <width>511</width>
     <height>99</height>
    </rect>
   </property>
   <property name="title">
    <string>Limit Sensors</string>
   </property>
   <widget class="QLabel" name="IPLabel_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Neg. Limit</string>
    </property>
   </widget>
   <widget class="QLabel" name="IPLabel_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Neg. SW Limit</string>
    </property>
   </widget>
   <widget class="QLabel" name="IPLabel_10">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Pos. SW Limit</string>
    </property>
   </widget>
   <widget class="QLabel" name="IPLabel_11">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Pos. Limit</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GroupBox_Sequences">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>511</width>
     <height>79</height>
    </rect>
   </property>
   <property name="title">
    <string>Sequences</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>40</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Sequence</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>79</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="checkButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Check</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="GroupBox_Motor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>510</width>
     <height>40</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QCheckBox" name="CheckBox_EnableMotor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Motor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Motor Type:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ComboBox_MotorType">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>79</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Servo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Stepper</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>30</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Axis: </string>
    </property>
   </widget>
   <widget class="QComboBox" name="ComboBox_AxisSelect">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>79</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>X</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Y</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Z</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
